From c349ed897313fa2ea4d4319ee16ece48455ae899 Mon Sep 17 00:00:00 2001
From: Sergey Efimochkin <efimochkin.sv@phystech.edu>
Date: Mon, 17 Dec 2018 04:14:33 +0300
Subject: [PATCH v2] staging: iio: adc: ad7280a: fix codestyle in ad7280_event_handler

staging: iio: adc: ad7280a: fix codestyle in ad7280_event_handler
Signed-off-by: Sergey Efimochkin <efimochkin.sv@phystech.edu>
---
Changes in v2:
- changed patch prefix
- changed patch description
- removed incorrect fixes to codestyle
 drivers/staging/iio/adc/ad7280a.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/drivers/staging/iio/adc/ad7280a.c b/drivers/staging/iio/adc/ad7280a.c
index 58420dcb406d..f167ee1b2127 100644
--- a/drivers/staging/iio/adc/ad7280a.c
+++ b/drivers/staging/iio/adc/ad7280a.c
@@ -725,8 +725,7 @@ static irqreturn_t ad7280_event_handler(int irq, void *private)
 		} else {
 			if (((channels[i] >> 11) & 0xFFF) >= st->aux_threshhigh)
 				iio_push_event(indio_dev,
-					       IIO_UNMOD_EVENT_CODE(
-							IIO_TEMP,
+					       IIO_UNMOD_EVENT_CODE(IIO_TEMP,
 							0,
 							IIO_EV_TYPE_THRESH,
 							IIO_EV_DIR_RISING),
@@ -734,8 +733,7 @@ static irqreturn_t ad7280_event_handler(int irq, void *private)
 			else if (((channels[i] >> 11) & 0xFFF) <=
 				st->aux_threshlow)
 				iio_push_event(indio_dev,
-					       IIO_UNMOD_EVENT_CODE(
-							IIO_TEMP,
+					       IIO_UNMOD_EVENT_CODE(IIO_TEMP,
 							0,
 							IIO_EV_TYPE_THRESH,
 							IIO_EV_DIR_FALLING),
-- 
2.15.1 (Apple Git-101)

